driver/tke_driver.go

```diff
@@ -296,36 +296,68 @@ func (d *Driver) GetDriverUpdateOptions(ctx context.Context) (*types.DriverFlags
 	driverFlag := types.DriverFlags{
 		Options: make(map[string]*types.Flag),
 	}
+	driverFlag.Options["secret-id"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "The secretID of the cluster",
+	}
+	driverFlag.Options["secret-key"] = &types.Flag{
+		Type:     types.StringType,
+		Password: true,
+		Usage:    "The secretKey of the cluster",
+	}
 	driverFlag.Options["cluster-id"] = &types.Flag{
 		Type:  types.StringType,
 		Usage: "The ID of the existing cluster",
 	}
-	driverFlag.Options["cluster-name"] = &types.Flag{
+	driverFlag.Options["goods-num"] = &types.Flag{
```

Comment:
I don't like the variable name `goods-num`. Can we rename it to `node-count` or something meaningful?